specification/dimensions/listunitprice.md

```diff
@@ -0,0 +1,31 @@
+# List Unit Price
+
+The List Unit Price represents the suggested provider published price for a single pricing measurement unit, which incorporates volume/tier-based reductions while excluding any negotiated or commitment based discounts. This price is denominated in the pricing currency. It is used to calculate List Cost and savings based on rate optimization activities.
+
+The ListUnitPrice column MUST be present in the billing data. This column MUST be a numeric value of type Decimal, within the range of valid positive decimal values and MUST NOT contain null values in case of usage-based charges.
+
+## Column ID
+
+ListUnitPrice
+
+## Display name
+
+List Unit Price
+
+## Description
+
+Represents the suggested provider published price for a single pricing measurement unit, which incorporates volume/tier-based reductions while excluding any negotiated or commitment based discounts.
+
+## Content Constraints
+
+| Constraint      | Value                            |
+|:----------------|:---------------------------------|
+| Column required | True                             |
+| Data type       | Decimal                          |
+| Allows nulls    | True                             |
```

Comment:
One of the members pointed out that **null** is probably more appropriate than '0' in certain cases (e.g. for chargeType: **tax**). As a result, we made the change to 'false' and underscored that '... MUST NOT contain null values in the case of usage-based charges.' 

**QUESTION**: Should we enforce this for both **usage-based and purchase-based** charges? 
_...MUST NOT contain null values for both usage-based and purchase-based charges_

Reply:
tend to agree. If we put null, aggregations won't be wrong. If we put 0 artificially, then we may screw up some numbers and make it unclear when the actual rate is 0. Whatever we decide, we need to be consistent with the other columns as well. 

The alternative is if the cost of tax is $100, we say quantity is 1 and unit is 'count' and rate is '100' type thing which could work though that rate is always moving and doesn't really mean anything.

Reply:
Since it's defined as the provider-published price, I prefer null in the case of tax-type charges. Still not sure if null should be allowed in case of purchase-based charges.

Reply:
The issue discussed and addressed in another conversation.